doc/source/getting_started/Introduction.ipynb

```diff
@@ -104,7 +104,7 @@
    "source": [
```

Comment:
<p>It is great to make these references to arviz functions link to their respective api page, but you should not do that with manual markdown links but with sphinx references instead. We have some docs on that at https://arviz-devs.github.io/arviz/contributing/developer_guide.html#documentation. In this case, you are referencing functions, so you should change the</p><p> </p><pre class="ql-syntax" spellcheck="false">[plot_posterior](https://arviz-devs.github.io/arviz/api/generated/arviz.plot_posterior.html)
</pre><p> </p><p>for</p><p> </p><pre class="ql-syntax" spellcheck="false">{func}~arviz.plot_posterior
</pre><p> </p><p>The same applies to all other references you are adding in this and future files. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>There are some extra <code>&lt;/span&gt;</code>  in the reference which I think will break them. Can you check locally that they work?</p><p> </p><p>Also note that after https://github.com/arviz-devs/arviz/pull/1778 you can now rebase on main to get previews of the documentation rendered for each PR. I think this will help us review future PRs</p>

Reply:
I checked the number of `<span` and `</span>` is the same in this file.

Reply:
when I look at reviewnb, I see `` {func}`~arviz.</span>plot_posterior` `` which is not a valid sphinx id. this pr has no readthedocs preview so I'm not sure if it's an nbviewer error or a mistake copy pasting from my earlier comment (copy pasting from reviewnb comments can be a nightmare). But it's something that needs to be double checked